src/en/charms-destroy.md

```diff
@@ -42,19 +42,23 @@ It is possible to remove individual units instead of the entire application
 (i.e. all the units):
 
 ```bash
-juju remove-unit mediawiki/1
+juju remove-unit postgresql/2
 ```
 
+In the case that the removed unit is the only one running on the corresponding
+machine, unless any of the following is true for that machine, it will also be
+removed:
```

Comment:
I get a little confused half-way through. Maybe something more like "the corresponding machine will be removed unless" ...

Reply:
:ok: